cypress/integration/tests/developer/applicationinventory/analysis/source_analysis.test.ts

```diff
@@ -48,6 +50,9 @@ describe("Source Analysis", { tags: "@tier1" }, () => {
         //Disable all proxy settings
         Proxy.disableAllProxies();
 
+        // Clears artifact repository
+        mavenConfiguration.clearRepository();
+
```

Comment:
application.analyze();
        application.verifyAnalysisStatus("Completed");
        application.openreport();
        application.validateStoryPoints();
- Small comment - I see those lines above repeated  all the time over different places - Why it is not a method ?!

Reply:
It absolutely should be a method, I will address it.

Reply:
I think it will be safer to address this comment in a different PR. 

This PR solves most of the errors of tier1 (and some of the tier 2) tests